message.go

```diff
@@ -131,6 +131,11 @@ func NewMessage(data []byte) *Message {
 	}
 }
 
+// GetLinkName returns associated link name or empty string if receiver or link is not defined,
+func (m *Message) GetLinkName() string {
```

Comment:
Do we actually need to expose this outside the SDK, i.e. is it sufficient to call `m.message.GetLinkName()` internally or are there scenarios where external callers need this?

Reply:
You are right. We can keep it package level function.